components/Icons.es6.js

```diff
@@ -1837,6 +1837,14 @@ function Spaceship ({ className, width="16", height="16", viewBox="0 0 16 16", c
   )
 }
```

Comment:
In the past, we have made 3 file changes for icons. Why only 1 in this case?

Reply:
Add the icon to the demo component 😄